webapp/src/components/PollDetailPage/PollProgress/PollProgress.tsx

```diff
@@ -37,7 +37,10 @@ class PollOption extends React.PureComponent<OptionProps> {
         <div className="mask">
           <div className="bg" />
           <span className="text">
-            {option.value}&nbsp;({percentage}%)
+            {option.value}
+            &nbsp;(
+            {percentage}
+            %)
```

Comment:
Nitpick: Maybe this could mimic https://github.com/decentraland/agora/pull/99/files#diff-f106eb8c0cdb9aef1be0a7d41323c78e so they both look the same (with or without `&nbsp;`